persistence-modules/java-jpa/src/test/java/com/baeldung/jpa/sqlresultsetmapping/SqlResultSetMappingUnitTest.java

```diff
@@ -44,7 +44,7 @@ public void whenNamedQuery_thenSingleEntityResult() {
 
     @Test
     public void whenNamedQuery_thenMultipleEntityResult() {
-        final Query query = em.createNativeQuery("SELECT e.id, e.name, d.id, d.employeeId, d.dayOfWeek "
+        final Query query = em.createNativeQuery("SELECT e.id as idEmployee, e.name, d.id as daysId, d.employeeId, d.dayOfWeek "
```

Comment:
why is this needed?

Reply:
It gives me an error to the column Id that is duplicated.